### DISCUSSION TITLE
Our school has never provided students with an official education email address, but the reason you refused to let me go was that I did not provide the school's designated email address.

### DISCUSSION THREAD

**Discussion by jsonza6** [community/community#53797] [GitHub Education]
我提供了我的录取通知书以及学生证进行学生认证，为什么还是频频被拒绝

**Comment by samus-aran** [comment]
Hi @zwq3107016156 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
